src/Microsoft.PowerShell.Commands.Utility/commands/utility/UtilityCommon.cs

```diff
@@ -59,35 +59,6 @@ public enum TextEncodingType
         Ascii,
     }
 
-    /// <summary>
-    /// Utility class to contain resources for the Microsoft.PowerShell.Utility module.
-    /// </summary>
-    public static class UtilityResources
```

Comment:
Seems we can not remove public APIs. We can use `obsolete` attribute.

Reply:
Is this is PowerShell standard?

Reply:
This type was there only because we used to have script functions that use the built-in resources files. This is theoretically a breaking change, but practically I don't think it will affect anyone :)

Reply:
A [search on GitHub](https://github.com/search?l=PowerShell&q=utilityresources&type=Code) returned a few instances of scripts using this public class.  Seems like there isn't a good reason to break them.

Reply:
We could move obsolete APIs (we have others) to separate dll and perhaps to separate compatibility package.

Reply:
For obsoleted types, you may be able to do that by type forwarding. For obsoleted type members, I don't think there is a way to do that. This should be a separate discussion.